package.json

```diff
@@ -33,6 +33,7 @@
     "js": "npm-run-all js-lint* js-compile* js-minify*",
     "js-main": "npm-run-all js-lint js-compile js-minify",
     "js-docs": "npm-run-all js-lint-docs js-minify-docs",
+    "js-lint-fix": "eslint --fix js/ && eslint --fix --config js/tests/.eslintrc.json --env node build/",
```

Comment:
I'm open for suggestions on how to name the command and if we can do some optimisations here.